### PR TITLE
Adding errata URL option

### PR DESCRIPTION
**Pull request by adammhaile**
@vikaslaad @smunilla @tbielawa @sosiouxme

### CODE REVIEW

File: jobs/build/release/Jenkinsfile [hunk 1]
```diff
@@ -42,6 +42,12 @@ node {
                         $class: 'hudson.model.StringParameterDefinition',
```
Comment (tbielawa) [inline jobs/build/release/Jenkinsfile]: Just gonna leave the old `ADVISORY` param in there for now?

Comment (smunilla) [reply jobs/build/release/Jenkinsfile]: I'd say yes. We can get this URL from the ET API eventually.

**Pull request merged**